src/components/sections/homepage/hero/index.tsx

```diff
@@ -355,56 +355,11 @@ export const Hero = () => {
               className: s.subtitle,
               children: (
                 <span>
-                  Replay is a time travel debugger here to make building web apps a breeze.
+                  Replay is a specialized browser and time travel debugger that lets you record, retroactively debug, and fix the hardest issues as a team.  Take control of your support process and test suite, so you can build faster and more efficiently.
```

Comment:
There's something here that we can test. The second sentence should be clearer though. There's also risk in leading with the term "time travel debugger".